sdk/textanalytics/Azure.AI.TextAnalytics/src/AuthenticationPolicy.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.TextAnalytics
+{
+    internal class AuthenticationPolicy : HttpPipelineSynchronousPolicy
```

Comment:
Let's give this a more descriptive name ... maybe SubscriptionKeyAuthenticationPolicy?